0.15 stuck in bootloop due to Irma scheduler error

```
time="2020-10-26T15:01:25Z" level=info msg="Initializing IRMA library..."
time="2020-10-26T15:01:25Z" level=info msg="irma baseurl: https://nuts-development.ons.io/auth/irmaclient"
time="2020-10-26T15:01:25Z" level=info msg="Updating schemes every 60 minutes"
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xaaac94]

goroutine 1 [running]:
github.com/jasonlvhit/gocron.(*Scheduler).Every(0x0, 0x3c, 0x15e23d8)
	/pkg/mod/github.com/jasonlvhit/gocron@v0.0.0-20191007145845-57f89394836a/gocron.go:486 +0x34
github.com/privacybydesign/irmago.(*Configuration).AutoUpdateSchemes(0xc0001c8690, 0x3c)
	/pkg/mod/github.com/privacybydesign/irmago@v0.5.1/irmaconfig.go:1304 +0xf1
github.com/privacybydesign/irmago/server.(*Configuration).verifyIrmaConf(0xc001096ea0, 0xc0001f4a80, 0xc0000ae558)
	/pkg/mod/github.com/privacybydesign/irmago@v0.5.1/server/conf.go:194 +0x9d
github.com/privacybydesign/irmago/server.(*Configuration).Check(0xc001096ea0, 0x475f9c, 0xc000108ba0)
	/pkg/mod/github.com/privacybydesign/irmago@v0.5.1/server/conf.go:99 +0x1ef
github.com/privacybydesign/irmago/server/irmaserver.New(0xc001096ea0, 0x4, 0x15c650f, 0x10)
	/pkg/mod/github.com/privacybydesign/irmago@v0.5.1/server/irmaserver/api.go:39 +0x32
github.com/nuts-foundation/nuts-auth/pkg.GetIrmaServer.func1()
	/pkg/mod/github.com/nuts-foundation/nuts-auth@v0.15.0/pkg/irmaconfig.go:91 +0x3aa
sync.(*Once).doSlow(0xc0001c2e60, 0xc0016b5a60)
	/usr/local/go/src/sync/once.go:66 +0xe3
sync.(*Once).Do(...)
	/usr/local/go/src/sync/once.go:57
github.com/nuts-foundation/nuts-auth/pkg.GetIrmaServer(0x15b9a5d, 0x6, 0x15c3b44, 0xe, 0xc000301760, 0x1f, 0xc000258c70, 0xe, 0xc000258cc0, 0x9, ...)
	/pkg/mod/github.com/nuts-foundation/nuts-auth@v0.15.0/pkg/irmaconfig.go:67 +0xf2
github.com/nuts-foundation/nuts-auth/pkg.(*Auth).Configure.func1()
	/pkg/mod/github.com/nuts-foundation/nuts-auth@v0.15.0/pkg/auth.go:146 +0x238
sync.(*Once).doSlow(0xc000120930, 0xc000027e68)
	/usr/local/go/src/sync/once.go:66 +0xe3
sync.(*Once).Do(...)
	/usr/local/go/src/sync/once.go:57
github.com/nuts-foundation/nuts-auth/pkg.(*Auth).Configure(0xc0001208c0, 0x0, 0x0)
	/pkg/mod/github.com/nuts-foundation/nuts-auth@v0.15.0/pkg/auth.go:128 +0x7d
github.com/nuts-foundation/nuts-go/cmd.configureEngines()
	/go/cmd/root.go:139 +0x5f
github.com/nuts-foundation/nuts-go/cmd.Execute()
	/go/cmd/root.go:93 +0x94
main.main()
	/go/main.go:25 +0x20
```

--- a/pkg/irmaconfig.go
+++ b/pkg/irmaconfig.go
@@ -43,16 +43,9 @@ func GetIrmaConfig(config AuthConfig) (irmaConfig *irma.Configuration, err error
 			return
 		}
 
-		if !config.SkipAutoUpdateIrmaSchemas {
-			logrus.Infof("Downloading irma schemas. If this annoys you or you want to pin the schemas, set %s", ConfSkipAutoUpdateIrmaSchemas)
-			if err = irmaConfig.DownloadDefaultSchemes(); err != nil {
-				return
-			}
-		} else {
-			logrus.Info("Loading irma schemas.")
-			if err = irmaConfig.ParseFolder(); err != nil {
-				return
-			}
+		logrus.Info("Loading irma schemas.")
+		if err = irmaConfig.ParseFolder(); err != nil {
+			return
 		}
 		_irmaConfig = irmaConfig
 	})

